modules/dnn/src/dnn.cpp

```diff
@@ -4459,7 +4459,7 @@ string Net::Impl::dump()
             prevNode = itBackend->second;
         }
     }
-    string colors[] = {"#ffffb3", "#fccde5", "#8dd3c7", "#bebada", "#80b1d3", "#fdb462", "#ff4848", "#b35151"};
+    string colors[] = {"#ffffb3", "#fccde5", "#8dd3c7", "#bebada", "#80b1d3", "#fdb462", "#ff4848", "#b35151", "#b662ff"};
```

Comment:
Does it make sense to replace to `std::vector<std::string>` and check `colorId < colors.size()`?

Reply:
btw, cppcheck found this here.

arent there static analyzers running over the code on the buildbots ?

Reply:
It's done time to time, but not for each PR.

Reply:
The check is good idea!

Reply:
so i'll add it ...